empress/xscape/eventscape.py

```diff
@@ -5,21 +5,16 @@
 
 # python libraries
 import time
+import csv
 from collections import *
```

Comment:
I found that removing this import passes all the provided script runs. However, I'm not 100% sure that this isn't use in the script. Do you want me to remove it and see if it will break at some point later? I think the error should be obvious when arose if existed. @ssantichaivekin 

This comment is as part of PR #54

Reply:
Sry, I pin the wrong line. I meant to pin line 9 ...

Reply:
You can also search the file whether the name `csv` is mentioned somewhere. If not remove it. Some IDEs such as PyCharm will tell you immediately when an import is unused.

Reply:
Jo, sry but I meant to pin `from collections import *` and I use VS Code without proper extension to track if any of the code in the file use anything from collections module.

Reply:
Oh my bad.

Reply:
I also think we should remove import *, but we can do that later :)